docs/en/operations/settings/settings.md

```diff
@@ -63,6 +63,26 @@ Default value: 0.
 </profiles>
 ```
 
+## dialect {#dialect}
```

Comment:
The "dialect" settings can be too generic for this. Maybe introduce another setting, `interval_output_format` with values `number` and `kusto`?

It will be consistent with other format-controlling settings.